parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -48,6 +48,9 @@ fi
 
 echo $(date),$(hostname), startcustomscript>>/opt/m
 
+# check cloud init status and log details, we already exit in cse_cmd.sh if cloud-init returns exit code 1, here we just log details of errors that cloud-init encountered
+handleCloudInitStatus "/var/log/azure/cluster-provision-cse-output.log";
```

Comment:
why do we need this if we're already exiting earlier? I assume these logs will already be present in the cloud-init logs

Reply:
Please pin the kusto query somewhere so that on-call people can refer to it when seeing CloudInit issue. Thanks for working on this PR! :)

Reply:
Will do, once I write it :')